### PR TITLE
Update store-apps-company-portal-autopilot.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document as fixed versions are valid for LOB apps, not offline apps

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/memdocs/issues/3300

File: memdocs/intune/apps/store-apps-company-portal-autopilot.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ To manage devices and install apps, your users can optionally use the Company Po
 
 ## Prerequisites
 
-You can choose to install the **Company Portal (Offline)** app using the steps below. The Company Portal app will be installed in device context when assigned to the Autopilot group and will be installed on the device before the user logs in. Offline apps are updated using Intune, whereas online apps are updated by the store. Use offline apps when you need to install and maintain a specific app version.
+You can choose to install the **Company Portal (Offline)** app using the steps below. The Company Portal app will be installed in device context when assigned to the Autopilot group and will be installed on the device before the user logs in. line-of-business (LOB) apps are updated using Intune, whereas online apps are updated by the store. Use LOB apps when you need to install and maintain a specific app version.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/store-apps-company-portal-autopilot.md:41]:
```suggestion
You can choose to install the **Company Portal (Offline)** app using the steps below. The Company Portal app will be installed in device context when assigned to the Autopilot group and will be installed on the device before the user logs in. Line-of-business (LOB) apps are updated using Intune, whereas online apps are updated by the store. Use LOB apps when you need to install and maintain a specific app version.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@Erikre Please review and Sign off this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by Erikre** [comment]
Offline apps are managed by Intune, whereas online apps are managed by the store. Use offline apps when you need to install and maintain a specific app version. A line-of-business (LOB) app is an app that you add to Intune from an app installation file. This kind of app is typically written in-house.

@msbemba  - Based on the above, I'm closing this PR. Let me know if you think we should re-open. Thanks

**Pull request closed without merging**